po/fr/minetest.po

```diff
@@ -19,6 +19,14 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=n > 1;\n"
 "X-Generator: Weblate 2.10-dev\n"
 
+#: builtin/client/chatcommands.lua
+msgid "Clear the out chat queue"
+msgstr "Nettoie la file d'attente de discussion en sortie"
+
+#: builtin/client/chatcommands.lua
+msgid "The out chat queue is now empty"
+msgstr "La file d'attente de discussion en sortie a été nettoyée"
+
```

Comment:
It will be overwritten by the translation from weblate sooner or later anyway, so I think you can keep it.